libs/components/lookup/testing/src/lookup/lookup-harness.spec.ts

```diff
@@ -14,7 +14,7 @@ async function setupTest(options: { dataSkyId: string }) {
   const fixture = TestBed.createComponent(LookupHarnessTestComponent);
   const loader = TestbedHarnessEnvironment.loader(fixture);
 
-  let lookupHarness: SkyLookupHarness;
+  let lookupHarness: SkyLookupHarness | null;
```

Comment:
How is it possible that `lookupHarness` is `null`? And if it is possible, could we not throw an error before returning since all the tests need it to not be `null`?

Reply:
The null check was because the `queryHarness` call on line 27 can return `null`. However, I just added a non-null there and removed the null checks elsewhere since this is in a test file and should be safe.